swagger/docs.go

```diff
@@ -1503,8 +1449,6 @@ var SwaggerInfo = &swag.Spec{
 	Description:      "APIs for Open Container Initiative Distribution Specification",
 	InfoInstanceName: "swagger",
 	SwaggerTemplate:  docTemplate,
-	LeftDelim:        "{{",
```

Comment:
Double check your swag version you used matches the one in the makefile, I suspect it doesn't.